test/tla/Rec11.tla

```diff
@@ -0,0 +1,33 @@
+--------------------------- MODULE Rec11 -------------------------------------
+(*
+ * A test for unfolding recursive definitions.
```

Comment:
notes here on what this is testing vs. the previous 10 `Rec`s might help in the future :)